pkg/enforce/enforce.go

```diff
@@ -318,6 +321,24 @@ func EnforceJob(ctx context.Context, ghc *ghclients.GHClients, d time.Duration,
 	}
 }
 
+// returns true if the repository has content and false if it's empty
+func isRepositoryEmptyReal(ctx context.Context, c *github.Client, owner, repo, policy string) (bool, error) {
+	opts := github.RepositoryContentGetOptions{}
+	file, dirs, resp, err := c.Repositories.GetContents(ctx, owner, repo, "", &opts)
+	if err != nil && resp.StatusCode != http.StatusNotFound {
+		return false, err
+	}
+	if file == nil && len(dirs) == 0 {
+		log.Info().
+			Str("org", owner).
+			Str("repo", repo).
+			Str("area", policy).
+			Msg("Repository is empty, skipping")
+	}
+
+	return true, nil
```

Comment:
Probably want to return false here, and true inside the if statement.

Reply:
Essential catch!